apps/odo/test/unit/odo/angry_case_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule Odo.AngryCaseTest do
+  use ExUnit.Case
+  use Placebo
+
+  import SmartCity.Event, only: [file_ingest_end: 0, error_file_ingest: 0]
+  alias ExAws.S3
+  alias SmartCity.HostedFile
+
+  test "Individual task failures do not stop others" do
+    start_supervised!(Odo.TestEventHandler)
+
+    # Application.put_env(:odo, :working_dir, "test/support")
+
+    File.cp!("test/support/my-data.shapefile", "/tmp/123.shapefile")
```

Comment:
do this tempdir work without using the `Temp` module to create a platform-independent tempdir that the app has permissions to write to? we had issues with trying to write to `/tmp` when we were moving the integration test to the e2e integration.

Reply:
Moved to use the Temp library